Sources/StreamChatUI/CommonViews/Attachments/AttachmentViews/ImageAttachmentComposerPreview.swift

```diff
@@ -41,7 +41,10 @@ open class ImageAttachmentComposerPreview: _View, ThemeProvider {
     
     override open func updateContent() {
         super.updateContent()
-        
-        imageView.loadImage(from: content, components: components)
+        components.imageLoader.loadImage(
```

Comment:
it seems we have the same issue here https://github.com/GetStream/stream-chat-swift/pull/1344/files#r694799931